### DISCUSSION TITLE
css/qss stylesheet for pyqtgraph

### DISCUSSION THREAD

**Discussion by patoutafay** [pyqtgraph/pyqtgraph#2800] [Q&A]
is it possible to use css/qss stylesheet on pyqtgraph element like plotwidget ?

**Comment by j9ac9k** [comment]
Yes; PlotWidget inherits from QGraphicsView, which inherits from QWidget.  

You can see some examples of the syntax here: https://doc.qt.io/qt-5/stylesheet-syntax.html

**Reply by patoutafay** [reply]
Hi, can you give me an example? I am not able to do it no matter what I've tried.
